Wikipedia/Code/MWKSection.m

```diff
@@ -275,6 +275,54 @@ - (NSString*)summary {
     }] componentsJoinedByString:@" "] wmf_summaryFromText];
 }
 
+static NSString* const WMFSectionDisambiguationTitlesXPathSelector = @"//div[@class='hatnote']//a/@href";
+
+- (nullable NSArray<MWKTitle*>*)disambiguationTitles {
+    NSArray* textNodes = [self elementsInTextMatchingXPath:WMFSectionDisambiguationTitlesXPathSelector];
+    if (!textNodes || !textNodes.count) {
```

Comment:
you only need to check `textNodes.count == 0`, checking `!textNodes` is implicitly done by checking the count.